RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cfi.py

```diff
@@ -73,5 +73,10 @@
       kPoorRecoFlagEE = cms.bool(False),
       chi2ThreshEB_ = cms.double(65.0),
       chi2ThreshEE_ = cms.double(50.0),
+
+      # for crossCorrelationMethod
+      crossCorrelationStartTime = cms.double(-25),
+      crossCorrelationStopTime = cms.double(25),
+      crossCorrelationTargetTimePrecision = cms.double(0.01),
```

Comment:
Is this the right place?

Reply:
Looks good, but I think you will also need to add the parameters to the fillDescription here https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc#L490.
Otherwise, it will give an exception at runtime.

Reply:
Indeed, I've added the required code.
However, while setting those parameters in the test python is changing the values in the algo, parameters here are not affecting them.

Reply:
Are they overridden by the fillDescription method?